README.md

```diff
@@ -12,167 +12,218 @@ Using Kubernetes primitives, administrators configure identities and bindings to
 
 ## Contents
 
-* [Getting Started](#getting-started)
-* [Demo](#demo)
+
 * [Components](#components)
-* [Features](docs/readmes/README.md#features)
```

Comment:
We need to keep the features. I think we can also expand and list out the features that are documented in the `docs/readmes/*` dir, wdyt?